synapse/cortex.py

```diff
@@ -1179,6 +1181,28 @@ async def initServiceStorage(self):
 
         await self.auth.addAuthGate('cortex', 'cortex')
 
+    async def getStormModsByIface(self, name):
```

Comment:
why is this async?

Reply:
Because every time i make a top level interface *not* async, we end up having to infect it eventually

Reply:
:smile: